vendor/hal_camera_default.te

```diff
@@ -35,3 +38,5 @@ r_dir_file(hal_camera_default, sysfs_esoc)
 
 allow hal_camera_default camera_vendor_data_file:dir rw_dir_perms;
 allow hal_camera_default camera_vendor_data_file:file create_file_perms;
+
+allow hal_camera_default kernel:system module_request;
```

Comment:
This is a `dontaudit` on crosshatch, so is it needed?

Reply:
@jerpelea wants allow all kernel module requests ... So allow it